addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -233,6 +319,56 @@ weSnippetEditor.SnippetsMenu.include({
             throw e;
         }
     },
+    /**
+     * @private
+     */
+    _onAnimateTextClick(ev) {
+        const sel = window.getSelection();
+        const editable = this.options.wysiwyg.$editable[0];
+        if (!sel.rangeCount) {
+            return;
+        }
+        const range = getDeepRange(editable, { splitText: true, select: true, correctTripleClick: true });
+        if (this.$currentAnimatedText.length) {
+            this.$currentAnimatedText.contents().unwrap();
+            this.options.wysiwyg.odooEditor.historyResetLatestComputedSelection();
+            this._toggleHighlightAnimatedTextButton();
+            ev.target.classList.remove('active');
+            this.options.wysiwyg.odooEditor.historyStep();
```

Comment:
Seems there is a historyStep after the if and an history step after the else... can't there be only one after the if..else ?

Reply:
no because in the else, if $snippet is null we dont need an history step

Reply:
Indeed, my bad :+1: